### ISSUE TITLE
Incorrect href when using `tel` Twig filter

### ISSUE THREAD

**Issue by dennisfrank** [rynpsc/craft-phone-number#2]
When using the `url` Twig filter, the site URL gets included in the `href` value:

```twig
{{ varWithPhoneNumber|tel('DE')
```

→ invalid result:

```html
<a href="http://example.test/tel:+49-123-4567879">+49 (0) 123 / 4567879</a>
```

expected:

```html
<a href="tel:+49-123-4567879">+49 (0) 123 / 4567879</a>
```

**Comment by rynpsc** [comment]
Which version off on the plugin are you using and which Craft version? There was an issue introduced in Craft 3.3 where they changed how URLs were created. This was fixed [1.2.4](https://github.com/rynpsc/craft-phone-number/releases/tag/1.2.4).

**Comment by rynpsc** [comment]
Just published 1.2.5 which fixes this.

**Comment by dennisfrank** [comment]
Thank you @rynpsc! 😃
